bignum.c

```diff
@@ -204,6 +204,20 @@ void bn_zero(bignum256 *a)
 	}
 }
 
+// sets a bignum to one.
+void bn_one(bignum256 *a)
```

Comment:
Since we are using `-std=gnu99`, I'd suggest making this and `bn_zero()` an `inline` function